packages/feedback-service/src/resolver.ts

```diff
@@ -21,7 +22,27 @@ export const FeedbackResolver = {
   Mutation: {
     addFeedback(root: any, args: any, ctx: any) {
       const data = new Feedback(args.input);
-      return data.save();
+      return data.save().then(response => {
+        Feedback.findById(response._id)
+        .then( fb =>{
+          if (fb){
+            const issue = {
+              "fields":{
+                "project":{
+                  "key": "ONEPLAT"
```

Comment:
Use environment variable for the project key as well.

Reply:
Updated, when i addfeedback from https://localhost:8080/graphql  its not reading environment variable, can you give me some suggestion ?

Reply:
Try checking the if the variable is defined.